### PR TITLE
Style 403 and 404 pages

### PR DESCRIPTION
**Pull request by aleksandraworhacz**
Style 404 and 403 error pages

### CODE REVIEW

File: templates/pages/errors/404.html [hunk 1]
```diff
@@ -1,14 +1,12 @@
 {{#partial "page"}}
 
-{{> components/common/breadcrumbs breadcrumbs=breadcrumbs}}
```
Comment (hectorfhurtado) [inline templates/pages/errors/404.html:3]: @aleksandraworhacz same as with 403 page, please leave the breadcrumbs in the page. Thank you!

File: templates/pages/errors/403.html [hunk 2]
```diff
@@ -3,14 +3,14 @@
 {{> components/common/breadcrumbs breadcrumbs=breadcrumbs}}
```
Comment (hectorfhurtado) [inline templates/pages/errors/403.html]: @aleksandraworhacz please enclose the page content in a `<div>` like in this other page: https://github.com/xfiveco/xcorner/blob/tailwind/templates/pages/category.html#L18-L20 for consistency sake :) 

Same for 404.html

Thank you.

File: templates/pages/errors/404.html (hunk 1 above)
Comment (aleksandraworhacz) [reply templates/pages/errors/404.html:3]: I wonder why do we need breadcrumbs on error pages? They look a bit off IMHO

Image 2023-09-08 at 1 02 17 PM

Comment (hectorfhurtado) [reply templates/pages/errors/404.html:3]: @aleksandraworhacz hmm, you're right. Breadcrumbs here don't adds here. Thank you

**Comment by hectorfhurtado** [comment]
@aleksandraworhacz please resolve conflicts and it's good for merging. Nice work

**Pull request closed without merging**